### PR TITLE
StartValidity set to beginning of integration period for DCS data

### PR DESCRIPTION
**Pull request by chiarazampolli**
@shahor02 , for TOF.

### CODE REVIEW

**Comment by chiarazampolli** [comment]
Errors do not seem related to the PR. @shahor02 @noferini , can you review it at your convenience?
Thanks in advance!

Chiara

**Review by shahor02 — APPROVED** [review]
looks fine for me, @noferini should I wait for your approval before merging?

**Comment by chiarazampolli** [comment]
Hello @noferini , 
Can you take a look at this one?
Thanks!

File: Detectors/TOF/calibration/testWorkflow/TOFDCSDataProcessorSpec.h
Comment (noferini) [inline Detectors/TOF/calibration/testWorkflow/TOFDCSDataProcessorSpec.h:117]: Hi Chiara,
does this creation time refer to the time provided by Pietro in the filename.
If yes, it is fine with me

**Comment by noferini** [comment]
Hi @chiarazampolli ,
I had only one question (probably trivial for you) added in my review.
The PR is ok.

**Comment by chiarazampolli** [comment]
Ciao @noferini , 
I am probably blind, but I don't see it... Can you point me at it?

File: Detectors/TOF/calibration/testWorkflow/TOFDCSDataProcessorSpec.h
Comment (chiarazampolli) [reply Detectors/TOF/calibration/testWorkflow/TOFDCSDataProcessorSpec.h:117]: This is the data points processing, where there is no file. The time here is when the data are sent by the proxy.

Comment (noferini) [reply Detectors/TOF/calibration/testWorkflow/TOFDCSDataProcessorSpec.h:117]: ok, i see. Thanks

**Review by noferini — APPROVED** [review]

**Pull request merged**
